### DISCUSSION TITLE
typeerror _b.authenticatorselection is undefined

### DISCUSSION THREAD

**Discussion by tradexsrl** [Ylianst/MeshCentral#5610] [Issues]
Hi , i'm trying to enable yubico hardware key but when i enter a name and click next i get this message

**Comment by si458** [comment]
From the looks of it, the web ui thinks ur browser doesn't support yubikey?
I sadly don't have a yubikey so I have no way of testing/fixing this for you!
Open a bug report and explain ur issue (follow the template and include all details please!)
Also don't forget to sponsor me so I can fix issues! https://github.com/si458 all links down left hand side
